### PR TITLE
ci: Undefined sanitizer

### PR DESCRIPTION
**Pull request by oleorhagen**

### CODE REVIEW

File: cmake/undefsan.cmake [hunk 1]
```diff
@@ -0,0 +1,7 @@
+include(cmake/helper.cmake)
+
+if (CMAKE_BUILD_TYPE STREQUAL "UndefSan")
+  sanitizer_add_compiler_flags(UNDEFSAN
+    "-fsanitize=undefined"
+    "-fsanitize=undefined")
```
Comment (vpodzime) [inline cmake/undefsan.cmake:6]: Maybe we should rename the function to `sanitizer_add_compiler_linker_flags()`? This looks a bit confusing otherwise.

**Review by vpodzime — APPROVED** [review]

File: cmake/undefsan.cmake (hunk 1 above)
Comment (oleorhagen) [reply cmake/undefsan.cmake:6]: Yeh, agreed. Good idea!

Comment (oleorhagen) [reply cmake/undefsan.cmake:6]: https://github.com/mendersoftware/mender/pull/1151/commits/73dc8de251eba249a343daa1c4309a9a2b3f32d0

Comment (oleorhagen) [reply cmake/undefsan.cmake:6]: Went for: `sanitizer_add_compiler_and_linker_flags()`

**Pull request merged**